spatialmath/sphere.go

```diff
@@ -200,3 +203,29 @@ func sphereInSphere(inner, outer *sphere) bool {
 func sphereInBox(s *sphere, b *box) bool {
 	return -pointVsBoxDistance(b, s.pose.Point()) >= s.radius
 }
+
+// ToPointCloud converts a sphere geometry into []r3.Vector.
+func (s *sphere) ToPointCloud(resolution float64) []r3.Vector {
+	// check for user defined spacing
+	var iter float64
+	if resolution != 0. {
```

Comment:
same thing here, explaining this behavior in the function comment would be helpful to someone peeking at the function